app/Main.hs

```diff
@@ -100,8 +100,23 @@ main = do
 
```

Comment:
Changing the commit message would be enough?

Reply:
No, because the commit would still contain changes that will need to be squashed against two different commits... 

It needs to be split. You can do this:

* `git reset --soft HEAD~1` to undo the latest commit and keep the changes in the working directory.
* Then stage related to the `Command` module, and run `git commit --fixup 15eff3`
* Then stage related to the tests, and run `git commit --fixup ade525`

Reply:
Done